### PR TITLE
Custom Translator has replaced Hub

### PR DESCRIPTION
**Pull request by Jann-Skotdal**

### CODE REVIEW

**Comment by Jann-Skotdal** [comment]
#sign-off

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointServer/administration/turn-on-automated-document-translation.md [hunk 1]
```diff
@@ -271,7 +271,7 @@ For more information, see [Set-SPTranslationServiceApplication](/powershell/modu
...
-- The Microsoft Translator Hub is an extension of Microsoft Translator and lets you build automatic language translation systems that integrate with your website. After you build a custom system, the **Test System** page on the **Projects** tab in the Microsoft Translator Hub displays a category ID. You can configure the Machine Translation Service to use the custom translation system by passing the category ID in the  `MachineTranslationCategory` parameter. For more information about the Microsoft Translator Hub, see [About Microsoft Translator Hub](https://go.microsoft.com/fwlink/?LinkId=330174).
+- Microsoft Custom Translator is a feature of Microsoft Translator and lets you achieve much higher adherence to the domain-specific terminology and style by training a custom translation system on previously translated in-domain documents. You can configure the Machine Translation Service to use the custom translation system by passing the category ID in the MachineTranslationCategory parameter. For more information about the Microsoft Custom Translator, see About [Microsoft Custom Translator](https://docs.microsoft.com/azure/cognitive-services/translator/custom-translator/overview).
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointServer/administration/turn-on-automated-document-translation.md:274]:
```suggestion
- Microsoft Custom Translator is a feature of Microsoft Translator and lets you achieve much higher adherence to the domain-specific terminology and style by training a custom translation system on previously translated in-domain documents. You can configure the Machine Translation Service to use the custom translation system by passing the category ID in the MachineTranslationCategory parameter. For more information about the Microsoft Custom Translator, see [What is Custom Translator?](https://docs.microsoft.com/azure/cognitive-services/translator/custom-translator/overview)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor suggestion.

**Comment by Jann-Skotdal** [comment]
Thanks @JohanFreelancer9 , I have committed the edit

**Comment by Jann-Skotdal** [comment]
#sign-off

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@SerdarSoysal Thank you for submitting feedback and contributing to the docs. We are currently investigating this.

**Pull request merged**